src/text-buffer.coffee

```diff
@@ -578,6 +578,7 @@ class TextBuffer
   #     that can be used to load the file's content.
   #   * `createWriteStream` A {Function} that returns a `Writable` stream
   #     that can be used to save content to the file.
+  #   * `existsSync` A {function} that returns the {Boolean}, true if the file exists, false otherwise
```

Comment:
Whoops, caught these on a last check:
* `{function}` should be `{Function}`
* Add a `.` on the end
* `returns the {Boolean}` -> `returns a {Boolean}`